# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/moz.build
# Commit: fa57574775b2
# Full Hash: fa57574775b29f4a03667d9c350a1be269adfaf7
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-09-26 21:35:42
# Regressor Bug: 1577643
# File Overlap Count: 1
# Description:
#   Bug 1577643 - Implement a security info class for the quic transport. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44073
# ==============================================================================

diff -r 02f72f7040e4 -r fa57574775b2 security/manager/ssl/moz.build
--- a/security/manager/ssl/moz.build	Thu Sep 26 10:14:53 2019 +0000
+++ b/security/manager/ssl/moz.build	Wed Sep 25 20:23:56 2019 +0000
@@ -59,6 +59,7 @@
 ]
 
 EXPORTS += [
+    'CommonSocketControl.h',
     'CryptoTask.h',
     'EnterpriseRoots.h',
     'nsClientAuthRemember.h',
@@ -74,6 +75,7 @@
     'RootCertificateTelemetryUtils.h',
     'ScopedNSSTypes.h',
     'SharedCertVerifier.h',
+    'TransportSecurityInfo.h',
 ]
 
 EXPORTS.mozilla += [
@@ -92,6 +94,7 @@
 ]
 
 UNIFIED_SOURCES += [
+    'CommonSocketControl.cpp',
     'ContentSignatureVerifier.cpp',
     'CryptoTask.cpp',
     'CSTrustDomain.cpp',